src/renderer/components/TabBar.js

```diff
@@ -51,6 +51,12 @@ const TabBar = ({ tabs, onIndexChange, defaultIndex = 0 }: Props) => {
   const tabRefs = useRef([]);
   const [index, setIndex] = useState(defaultIndex);
 
+  const [mounted, setMounted] = useState(false);
+
+  useEffect(() => {
+    setMounted(true);
+  }, []);
```

Comment:
we'll need a more elegant solution to prevent the refs concept that force us to do this double rendering.